README.md

```diff
@@ -27,8 +27,9 @@ The package manager is bundled with the [downloads available at swift.org](https
 
 If you want to verify you already have the package manager installed, enter the following in a terminal:
 
-    swift build --help
-
+```bash
```

Comment:
How about using `sh` instead of `bash` since there is nothing bash-specific here